src/store/modules/AuthModule.js

```diff
@@ -31,7 +31,7 @@ class AuthModule extends VuexModule {
   cookieExpireChecker = new NonReactive(new CookieExpireChecker());
 
   /** @type {CHALLENGE_TYPES[keyof CHALLENGE_TYPES]?} */
-  challengeType = null;
+  challengeType = CHALLENGE_TYPES.EMAIL_OTP;
```

Comment:
Are you sure, that if we will have slow network or anything else we would not see `email` form before the all requests fulfilling?

Reply:
`challengeType` is used when passing email, so no need to check for slow network and etc....